qt/scientific_interfaces/ISISReflectometry/test/Batch/RowProcessingAlgorithmTest.h

```diff
@@ -293,7 +294,7 @@ class RowProcessingAlgorithmTest : public CxxTest::TestSuite {
   PreviewRow makePreviewRow(double theta = 0.1, const std::string &processingInstructions = "10-11") {
     auto previewRow = PreviewRow({"12345"});
     previewRow.setTheta(theta);
-    previewRow.setProcessingInstructions(processingInstructions);
+    previewRow.setProcessingInstructions(ROIType::Signal, processingInstructions);
```

Comment:
It would be good to add some tests to check that background and transmission processing instructions get set from a PreviewRow